### DISCUSSION TITLE
I've created a theme + server + Docker image that allows me to edit the picasa.ini file via an API

### DISCUSSION THREAD

**Discussion by uglow** [thumbsup/thumbsup#282] [Show and tell]
Hi @rprieto , 

I guess this is more of brief walkthrough of what is possible with ThumbsUp: I made it possible to change the picasa-star value while browsing the gallery. See https://github.com/uglow/thumbsup-fav-server

It is a bit of a Frankenstein at the moment, as the "theme" also contains the Express server (to listen for star-toggle requests) and it has a Docker image which extends from the ThumbsUp Docker image. The README hopefully provides some idea of how to install it on a Synology box (my use-case), but it is ultimately just a Docker image with a server listening on port 8080.

Anyway, I thought I'd let you know about it. I don't have a lot of time to work on open-source things, but this is a passion-project for me as I use it for my own photo library. Next things to add is a QR code generator button in the detail-view, so that I can link a paper-based photobook to related videos / galleries on my local webserver.

Thanks for writing ThumbsUp - it has saved me countless hours of building something myself, and saved me from going down the dreaded PHP 🤮  path.

Cheers, 
Brett

**Comment by rprieto** [comment]
Hi @uglow. I'm only seeing your post now, that's a very cool idea! Thanks for making it available on Github. I don't think it would belong in the "core" of thumbsup because it will always stay CLI-only (no backend). However it definitely fills a gap (managing favourites) that I've also wondered about. I'm sure many people will find your image / approach useful.
